### PR TITLE
Fix compiler warnings

### PR DESCRIPTION
**Pull request by ptschip**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -95,6 +95,9 @@ map<uint256, COrphanTx> mapOrphanTransactions GUARDED_BY(cs_orphancache);
 map<uint256, set<uint256> > mapOrphanTransactionsByPrev GUARDED_BY(cs_orphancache);
 void EraseOrphansFor(NodeId peer) EXCLUSIVE_LOCKS_REQUIRED(cs_orphancache);
 
+// BU: start block download at a low number in case our peers are slow when we start  
+static unsigned int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 1;       
+static unsigned int BLOCK_DOWNLOAD_WINDOW = 8;
```
Comment (gandrewstone) [inline src/main.cpp]: I know that its weird that all these constants are defined in main.h.  I would not have done it that way.  However, there is value in keeping all the constants together as much as possible, and in not moving things around for ease of rebase.  So please keep these in their original location in main.h :-(

**Comment by ptschip** [comment]
these are no longer constants, that's why i had to take them out of
main.h, because it gives so many warnings.  The originals are still in
main.h although commented out so it shouldn't affect any rebase.

**Comment by gandrewstone** [comment]
ok I see what you mean by with all of the unused variable warnings, so you're right we do need to split these off from main.h.  Can you copy the variable description comments over as well so we don't lose them?

**Comment by ptschip** [comment]
ok, it's done

**Comment by gandrewstone** [comment]
thanks!

**Pull request merged**